docs/pages/connect-your-client/web-ui.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Using the Web UI
+description: Using the Teleport Web UI
+---
+The Teleport Web UI is a web-based visual interface from which you can access resources, 
+view active sessions and recordings, create and review Access Requests, 
+manage users and roles, and more.
+
+This page serves a reference on certain Web UI features and their usage.
+
+## Joining an active session
+
+The Web UI allows you to list and join active SSH sessions via a web-based terminal.
+
+Any active SSH sessions that you are allowed to list will be listed on the "Active Sessions" page, which can be 
+accessed from the navigation bar on the left side. If you don't see the "Active Sessions" tab, it means that your user's role doesn't grant
+you `list` access for the `ssh_session` resource. Please refer to the [Teleport Access Controls Reference](../../access-controls/reference.mdx)
```

Comment:
>content/docs/pages/connect-your-client/web-ui.mdx
  17:71-17:144  warning  Link to unknown file: `../../access-controls/reference.mdx`. Did you mean `../access-controls/reference.mdx`  missing-file  remark-validate-links